test/acceptance_with_go_client/bm25_test.go

```diff
@@ -29,11 +30,53 @@ var paragraphs = []string{
 	"this has nothing to do with the rest",
 }
 
+var (
+	TRUE = true
+	ctx  = context.Background()
+)
+
+func AddClassAndObjects(t *testing.T, className string, datatype string, c *client.Client) {
+	class := &models.Class{
+		Class: className,
+		Properties: []*models.Property{
+			{Name: "contents", DataType: []string{datatype}, Tokenization: "word", IndexInverted: &TRUE},
```

Comment:
since `IndexInverted` is deprecated now, use `IndexFilterable` and `IndexSearchable` instead